src/workspace/workspace.es6

```diff
@@ -267,9 +267,10 @@ export const init = (parent) => {
    parent.append(root);
    rv.bind(root[0], state);
 }
-export const addDialog = (name, clickCb, removeCb) => {
+export const addDialog = (name, id, clickCb, removeCb) => {
```

Comment:
Why did you add `id` as the second argument to this function?
Are we passing the value once we invoke the function?

Reply:
we need this id for removeDialag function. yes we are.

Reply:
Where do we invoke the function from?

Reply:
in windows.es6 file line 373